src/patternfly/components/Page/page.scss

```diff
@@ -422,6 +422,10 @@
   }
 }
 
+.pf-c-page__sidebar + .pf-c-drawer {
```

Comment:
Were we going to look at using `.pf-c-page__drawer` here? Seems like we would still want to maintain the other page styles that are in the `.pf-c-page__main, .pf-c-page__drawer` selector, and for the ability to use page sections and things like that.